tests/slash-menu.spec.ts

```diff
@@ -1,78 +1,124 @@
-import { expect, test } from '@playwright/test';
+import { expect, Locator, Page, test } from '@playwright/test';
+import { SHORT_KEY } from 'utils/actions/keyboard.js';
 import {
   enterPlaygroundRoom,
   focusRichText,
   initEmptyParagraphState,
 } from 'utils/actions/misc.js';
-import { assertStoreMatchJSX } from 'utils/asserts.js';
+import {
+  assertAlmostEqual,
+  assertRichTexts,
+  assertStoreMatchJSX,
+} from 'utils/asserts.js';
 
-test('slash menu should show and hide correctly', async ({ page }) => {
-  await enterPlaygroundRoom(page, { enable_slash_menu: true });
-  const { paragraphId } = await initEmptyParagraphState(page);
-  await focusRichText(page);
-  const slashMenu = page.locator(`.slash-menu`);
+test.describe('slash menu should show and hide correctly', () => {
+  // See https://playwright.dev/docs/test-retries#reuse-single-page-between-tests
+  test.describe.configure({ mode: 'serial' });
```

Comment:
This pattern is great! I thinks we can migrate pretty much existing tests into this approach, so as to eliminate the cost starting new browser tab in each new spec. PRs welcome 😀